Core/COObject.m

```diff
@@ -1305,6 +1323,9 @@ - (void)validateNewValue: (id)newValue
 	ETModelDescriptionRepository *repo = _objectGraphContext.modelDescriptionRepository;
 	BOOL isPersistentRelationship = propertyDesc.isPersistentRelationship;
 	BOOL isValidatableType = ([self serializationGetterForProperty: propertyDesc.name] == NULL);
+	
+	if (!isValidatableType)
+		return;
```

Comment:
Won't this skip a bunch of the ETAsserts below? maybe that could hide bugs in the future.